File: fibonacci_series/C#/fibonacci_recursive.cs
using System;
public static class FibonacciRecursive{

public static void Main(){
    Console.Write("Enter the number of terms in the fibonacci sequence : ");
    int len= Convert.ToInt32(Console.ReadLine());
    Fibonacci_Recursive(len);
}    

public static void Fibonacci_Recursive(int len )  
{  
   Fibonacci_Rec_Temp(0, 1, 1, len);  
}  
private static void Fibonacci_Rec_Temp(int a, int b, int counter, int len)  
{  
    if (counter <= len)  
    {  
        Console.Write("{0} ", a);  
        Fibonacci_Rec_Temp(b, a + b, counter+1, len);  
    }  
}  
}

File: fibonacci_series/Swift/fibonacci-recursive.swift
func fibonacci(n:Int)->Int{
	if (n==0) || (n==1){
		return n  //In case n==0 or n==1, we don't always print the first two elements
	}
	else
	{
		return fibonacci(n:n-1) + fibonacci(n:n-2)
	}
}

print("Enter the number of elements:")
let number = readLine()
var num=Int(number!)
var i=0
while(num!>=i){
	print(fibonacci(n:i))
	i+=1
}